domain/learningandworkprogress/src/main/kotlin/uk/gov/justice/digital/hmpps/domain/learningandworkprogress/induction/InductionSchedule.kt

```diff
@@ -51,24 +51,25 @@ enum class InductionScheduleCalculationRule(val existingPrisonerWhenScheduleCrea
   EXISTING_PRISONER_UN_SENTENCED(true),
 }
 
-enum class InductionScheduleStatus(val inScope: Boolean, val isExclusion: Boolean = false, val isExemption: Boolean = false) {
-  SCHEDULED(true),
-  COMPLETED(true),
-  EXEMPT_PRISONER_DRUG_OR_ALCOHOL_DEPENDENCY(false, isExclusion = true),
-  EXEMPT_PRISONER_OTHER_HEALTH_ISSUES(false, isExclusion = true),
-  EXEMPT_PRISONER_FAILED_TO_ENGAGE(false, isExemption = true),
-  EXEMPT_PRISONER_ESCAPED_OR_ABSCONDED(false, isExemption = true),
-  EXEMPT_PRISONER_SAFETY_ISSUES(false, isExclusion = true),
-  EXEMPT_PRISON_REGIME_CIRCUMSTANCES(false, isExclusion = true),
-  EXEMPT_PRISON_STAFF_REDEPLOYMENT(false, isExemption = true),
-  EXEMPT_PRISON_OPERATION_OR_SECURITY_ISSUE(false, isExemption = true),
-  EXEMPT_SECURITY_ISSUE_RISK_TO_STAFF(false, isExclusion = true),
-  EXEMPT_SYSTEM_TECHNICAL_ISSUE(false), // system down
-  EXEMPT_PRISONER_TRANSFER(false, isExemption = true),
-  EXEMPT_PRISONER_RELEASE(false, isExemption = true),
-  EXEMPT_PRISONER_DEATH(false, isExemption = true),
-  EXEMPT_SCREENING_AND_ASSESSMENT_IN_PROGRESS(false, isExemption = true),
-  EXEMPT_SCREENING_AND_ASSESSMENT_INCOMPLETE(false, isExemption = true),
+enum class InductionScheduleStatus(val isExclusion: Boolean = false, val isExemption: Boolean = false) {
+  PENDING_INITIAL_SCREENING_AND_ASSESSMENTS_FROM_CURIOUS,
```

Comment:
I've added this new schedule status (PENDING_INITIAL_SCREENING_AND_ASSESSMENTS_FROM_CURIOUS), specifically to support PES
Whilst we dont need to do PES yet, I wanted to get the process straight in my head including the basic PES implementation.

I've also removed the `inScope` property as we dont use it and most likely wont. It was added initially because we thought being in or out of scope was something we were interested in, but we now know the concept of being in or out of scope is more relevant to Curious.